TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/SocketIOTpService.java

```diff
@@ -35,12 +36,14 @@ private SocketIOTpService() {
     public Observable<Result> connectTo(String serverAddress) {
         try {
             URL url = new URL(serverAddress);
-            socket = IO.socket(url.toExternalForm());
+            IO.Options options = new IO.Options();
+            options.query = ClientType.BOT.rawClientType();
```

Comment:
Pass a query string rather than creating separate events for the human and bot connection.